lib/chat_api/customers.ex

```diff
@@ -196,6 +213,7 @@ defmodule ChatApi.Customers do
     end
   end
 
+  @spec remove_tag(Customer.t(), binary()) :: {:ok, Customer.t()} | {:error, Ecto.Changeset.t()}
```

Comment:
i forget exactly what `Repo.delete()` returns, but it should be a tag or maybe a number? (but not `Customer.t()`)

Reply:
https://hexdocs.pm/ecto/Ecto.Repo.html#c:delete/2 it returns
```elixir
delete(
  struct_or_changeset :: Ecto.Schema.t() | Ecto.Changeset.t(),
  opts :: Keyword.t()
) :: {:ok, Ecto.Schema.t()} | {:error, Ecto.Changeset.t()}
```

Reply:
It will return back deleted schema

Reply:
Ahhh, you right in here, there is a nesting stuff, and it will return `CustomerTag` because it loads it through `get_tag(tag_id)`